tools/RepoTasks/RepoTasks.RemoteWorker/AppDomainWorker.cs

```diff
@@ -161,61 +197,205 @@ internal Configuration BuildXmlConfiguration(IEnumerable<Type> cmdletTypList)
 
-                if (ps1XmlAttribute == null ||
-                    (ps1XmlAttribute.Target & ViewControl.Table) != ViewControl.None)
+                        var listItem = (ps1XmlAttribute.ScriptBlock != null)
+                            ? new ListItem
+                            {
+                                ScriptBlock = ps1XmlAttribute.ScriptBlock,
+                            }
+                            : new ListItem
+                            {
+                                PropertyName = memberInfo.Name,
+                            };
+
+                        if (position == Ps1XmlConstants.DefaultPosition)
+                        {
+                            listItems.Add(listItem);
+                        }
+                        else
+                        {
+                            listItemsWithPosition.Add(Tuple.Create(listItem, position));
+                        }
+                    }
+                }
+                else
                 {
                     tableHeaders.Add(new TableColumnHeader
                     {
-                        Label = ps1XmlAttribute?.Label ?? memberInfo.Name
+                        Label = memberInfo.Name
                     });
 
                     tableColumnItems.Add(new TableColumnItem
                     {
                         Alignment = Alignment.Left,
                         PropertyName = memberInfo.Name
                     });
-                }
 
-                if (ps1XmlAttribute == null ||
-                    (ps1XmlAttribute.Target & ViewControl.List) != ViewControl.None)
-                {
                     listItems.Add(new ListItem
                     {
                         PropertyName = memberInfo.Name
                     });
                 }
             }
+
+            // track duplicate
+            if (dupTrackerGroupBy.Count > 1)
+            {
+                var warning = "Found a duplicate of property'GroupByThis'. 'GroupByThis=true' have the following properties: ";
+
+                foreach (var tuple in dupTrackerGroupBy)
+                {
+                    var outputTypeName = tuple.Item1;
+                    var baseTypeName = tuple.Item2;
+                    var propName = tuple.Item3;
+                    warning = warning + "'" + propName + "' in the class " + outputTypeName + " : " + baseTypeName + "; ";
+                }
+
+                warnings.Add(warning); 
+            }
+
+            var duplicates = dupTrackerPosition
+                .GroupBy(p => p.Item4)
+                .Where(g => g.Count() > 1)
+                .ToList();
+
+            if (duplicates.Count > 0)
+            {
+                var warning = "Found a duplicate of property'Position'. ";
+                foreach (var duplicate in duplicates)
+                {
+                    warning += $"'Positon={duplicate.Key}' have the following properties:";
```

Comment:
Nitpick : What do you think of the wording :  _Following properties had the same `Position` value of `{duplicate.key} specified:`"_

Reply:
fixed